homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/gravatar/GravatarViewModel.kt

```diff
@@ -197,7 +198,7 @@ internal class GravatarViewModel(
             _uiState.update { currentState ->
                 currentState.copy(
                     isRefreshing = isRefreshing,
-                    isLoading = !isRefreshing
+                    isLoading = !isRefreshing || currentState.avatars == null,
```

Comment:
I think we should show the loading after pull to refresh in case the avatars are null. WDYT?